### DISCUSSION TITLE
Is it normal to receive huge gaps in public timeline that says "Filtered" when I have "Hide completely" selected for my filter?

### DISCUSSION THREAD

**Discussion by Diwannist** [mastodon/mastodon#32775] [Q&A] [answered]
![Screenshot 2024-11-05 at 8 29 55 PM](https://github.com/user-attachments/assets/f005e94b-14ee-4e35-808a-630a93c96e49)
![IMG_7380](https://github.com/user-attachments/assets/57ad495a-b9c0-41ea-8318-6618217aee87)

**Comment by trwnh** [comment, marked as answer]
no, i think you should report an issue over at https://github.com/mastodon/mastodon-ios/issues/
